.circleci/config.yml

```diff
@@ -95,9 +95,7 @@ workflows:
       - test:
           requires:
             - build
-      - applitools:
-          requires:
-            - build
+      - applitools
```

Comment:
Making this run in parallel to build as it might be long-running, also we get parallel jobs now 🎉